xen/arch/arm/domain.c

```diff
@@ -44,6 +44,9 @@
 
 DEFINE_PER_CPU(struct vcpu *, curr_vcpu);
 
+#define ARMV8_PMU_PMCR_N_SHIFT 11
```

Comment:
AFAIK, this should go somewhere in `processor.h`/`cpu.h`, where rest of registers bits are defined

Reply:
Thank you for your review. I moved defines to `asm/arm64/sysregs.h` because as I see, `processor.h` and `cpu.h` don't contain such kind of defines.